patchalerts/games/factorio.py

```diff
@@ -38,9 +38,18 @@ def get_description(self, _url):
             if item.name is None:
                 continue
 
+<<<<<<< HEAD
+            if item.name == "strong": 
```

Comment:
Thanks for the additions, but it looks like there are some merge issues pending here - mind updating your branch first?